package.json

```diff
@@ -308,8 +308,11 @@
     "ci-backend": "clojure -X:dev:ee:ee-dev:drivers:drivers-dev:eastwood && clojure -X:dev:test",
     "test-cypress-run": "node ./frontend/test/__runner__/run_cypress_tests.js",
     "test-cypress-open": "./bin/build-for-test && yarn test-cypress-run --e2e --open",
+    "test-cypress-open-qa": "yarn test-qa-dbs:up && ./bin/build-for-test && QA_DB_ENABLED=true yarn test-cypress-run --e2e --open",
```

Comment:
Does this work?
`yarn test-qa-dbs:up && QA_DB_ENABLED=true yarn test-cypress-open`?